fixture-monkey/src/main/java/com/navercorp/fixturemonkey/FixtureMonkeyBuilder.java

```diff
@@ -587,4 +575,37 @@ public FixtureMonkey build() {
 			registeredPriorityMatchersByName
 		);
 	}
+
+	private void applyDeferredOptions() {
+		if (defaultPropertyNameResolver != null) {
+			fixtureMonkeyOptionsBuilder.defaultPropertyNameResolver(defaultPropertyNameResolver);
+		}
+
+		for (MatcherOperator<PropertyNameResolver> propertyNameResolver : propertyNameResolvers) {
+			fixtureMonkeyOptionsBuilder.insertFirstPropertyNameResolver(propertyNameResolver);
+		}
+	}
+
+	private MonkeyExpressionFactory buildExpressionFactory(FixtureMonkeyOptions fixtureMonkeyOptions) {
+		if (!expressionStrictMode) {
+			return this.monkeyExpressionFactory;
+		}
+
+		PropertyNameResolver compositePropertyNameResolver = buildCompositePropertyNameResolver(fixtureMonkeyOptions);
+
+		return new StrictModeMonkeyExpressionFactory(
+			new ArbitraryExpressionFactory(),
+			compositePropertyNameResolver
+		);
+	}
+
+	private PropertyNameResolver buildCompositePropertyNameResolver(
+		FixtureMonkeyOptions fixtureMonkeyOptions
+	) {
+		return property -> fixtureMonkeyOptions.getPropertyNameResolvers().stream()
+			.filter(it -> it.getMatcher().match(property))
+			.findFirst()
+			.map(it -> it.getOperator().resolve(property))
+			.orElseGet(() -> fixtureMonkeyOptions.getDefaultPropertyNameResolver().resolve(property));
+	}
```

Comment:
To keep the consistency, it would be better to use the prefix of `new` instead of `build`.

This could cause confusion in the context of the FixtureMonkeyBuilder.